test/box/gh-5422-broken_snapshot.test.lua

```diff
@@ -49,7 +49,7 @@ end;
 test_run:cmd("setopt delimiter ''");
 
 garbage_size = 500
-corruption_offset = 15000
+corruption_offset = 5000
```

Comment:
Why did you change offset? In commit message you are saying that we are about only to remove one check...